spyder_profiler/widgets/profilergui.py

```diff
@@ -351,6 +352,26 @@ def show_data(self, justanalyzed=False):
                                                time.localtime())
         self.datelabel.setText(date_text)
 
+def gettime_s(text):
+    pattern = '([+-]?\d+\.?\d*) ?([munsecinh]+)'
+    matches = re.findall(pattern, text)
+    if len(matches) == 0:
+        return None
+    time = 0.
+    for res in matches:
+        tmp = float(res[0])
+        if res[1] == 'ns':
+            tmp*=1e-9
+        elif res[1] == 'us':
+            tmp*=1e-6
+        elif res[1] == 'ms':
+            tmp*=1e-3
+        elif res[1] == 'min':
+            tmp*=60
+        elif res[1] == 'h':
+            tmp*=3600

```

Comment:
@impact27 thanks for working on this.

Could you include an space before and after `*=` so that all branches go from
`tmp*=3600` to `tmp *= 3600`